tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_InferredAssetSQLExample.ipynb

```diff
@@ -0,0 +1,1215 @@
+            },
+            "text/plain": [
+              "Calculating Metrics:   0%|          | 0/9 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/plain": [
+              "{\n",
+              "  \"success\": true,\n",
+              "  \"expectation_config\": {\n",
+              "    \"kwargs\": {\n",
+              "      \"column\": \"trip_distance\",\n",
+              "      \"min_value\": 1.6,\n",
+              "      \"max_value\": 1.6,\n",
+              "      \"strict_min\": false,\n",
+              "      \"strict_max\": false\n",
+              "    },\n",
+              "    \"meta\": {\n",
+              "      \"auto_generated_at\": \"20220908T014809.543396Z\",\n",
+              "      \"great_expectations_version\": \"0.15.21+28.g75ae687f9.dirty\"\n",
+              "    },\n",
+              "    \"expectation_type\": \"expect_column_median_to_be_between\"\n",
+              "  },\n",
+              "  \"exception_info\": {\n",
+              "    \"raised_exception\": false,\n",
+              "    \"exception_traceback\": null,\n",
+              "    \"exception_message\": null\n",
+              "  },\n",
+              "  \"meta\": {},\n",
+              "  \"result\": {\n",
+              "    \"observed_value\": 1.6\n",
+              "  }\n",
+              "}"
+            ]
+          },
+          "execution_count": 21,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "validator.expect_column_median_to_be_between(column=\"trip_distance\", auto=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "c6a6c277",
+      "metadata": {},
+      "source": [
+        "The `auto=True` will also automatically run the Expectation against the most recent Batch (which has an observed value of `1.61`) and the Expectation will pass. "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "90b8b938",
+      "metadata": {},
+      "source": [
+        "You can now save the `ExpectationSuite`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 22,
+      "id": "eba880ed",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "validator.save_expectation_suite()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "1b7ec631",
+      "metadata": {},
+      "source": [
+        "### Running the `ExpectationSuite` against single `Batch`"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "477381f5",
+      "metadata": {},
+      "source": [
+        "Now the ExpectationSuite can be used to validate single batches using a Checkpoint. In our example, let's validate a different table, `yellow_tripdata_sample_2020_02`, using the `ExpectationSuite` we built from `yellow_tripdata_sample_2020_01`."
```

Comment:
Not sure where these tables are coming from - I thought we only had one table `yellow_tripdata_sample_2020` and we are pulling different sets of batches from that?